### ISSUE TITLE
Uninstalling 1.1.0-preview1 runtime breaks 1.0.1 runtime - Fix in 1.1.x

### ISSUE THREAD

**Issue by rakeshsinghranchi** [dotnet/core-setup#1575]
## Steps to reproduce

1. Install [1.0.1 runtime](https://go.microsoft.com/fwlink/?LinkID=827515)
(I see that the `C:\Program Files\dotnet\host\fxr\1.0.1` directory gets created)
2. Install [1.1.0-preview1 runtime](https://go.microsoft.com/fwlink/?LinkID=831444)
3. Uninstall 1.1.0-preview1 runtime
(There is no longer a `C:\Program Files\dotnet\host` directory)
## Expected  behavior
```
$ dotnet

Microsoft .NET Core Shared Framework Host

  Version  : 1.0.1
  Build    : cee57bf6c981237d80aa1631cfe83cb9ba329f12

Usage: dotnet [common-options] [[options] path-to-application]

Common Options:
  --help                           Display .NET Core Shared Framework Host help.
  --version                        Display .NET Core Shared Framework Host version.

Options:
  --fx-version <version>           Version of the installed Shared Framework to use to run the application.
  --additionalprobingpath <path>   Path containing probing policy and assemblies to probe for.

Path to Application:
  The path to a .NET Core managed application, dll or exe file to execute.

If you are debugging the Shared Framework Host, set 'COREHOST_TRACE' to '1' in your environment.

To get started on developing applications for .NET Core, install .NET SDK from:
  http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
```

## Actual behavior
```
$ dotnet
A fatal error occured, the required library hostfxr.dll could not be found at C:\Program Files\dotnet
```

## Environment data
`dotnet --info` output:
```
A fatal error occured, the required library hostfxr.dll could not be found at C:\Program Files\dotnet
```

**Comment by rakeshsinghranchi** [comment]
Refer similar fix in 2.0.0 : https://github.com/dotnet/core-setup/pull/1571#pullrequestreview-23143709

**Comment by vivmishra** [comment]
Fixed and merged.
